src/data/issue_extraction.ipynb

```diff
@@ -2,19 +2,13 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            key = f"data/{savename}.csv"
</pre><p>can you add an additional directory name to this key so that its clear what project in the shared bucket the data is associated with? "github-labeler/data/{savename}.csv" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>